client/src/components/Feed/NestedComments.js

```diff
@@ -149,8 +149,7 @@ const NestedComments = ({
 
   const handleSubmit = async () => {
     let response;
-    const postId = comment.postId;
-    const commentId = comment._id;
+    const { postId , commentId } = comment;
```

Comment:
```suggestion
    const { _id: commentId, postId } = comment;
```

`commentId` is undefined otherwise so you need to rename in destructuring

![image](https://user-images.githubusercontent.com/10546720/86944252-7a62f100-c115-11ea-8595-9614006b234c.png)